### ISSUE TITLE
[Accessibility]A11y_VSCode_LangMode_ColorContrast: Color Contrast ratio is less than 4.5:1 for the placeholder in "LangMode"

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52541]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
ColorContrast,MAS1.4.3

**Repro Steps:**
1.Navigate to Status Bar and select "Select Language Mode" button.
2.Open color contrast analyzer
3.Observed that,color contrast ratio is less than 4.5:1 ratio.

**Actual:**
Checking through color contrast analyzer, the color contrast ratio for the placeholder "Select Language Mode" is 2.2:1

**Expected:**
The color contrast ratio should be equal to or greater that 4.5:1, here the placeholder should be equal or greater that 4.5:1 ratio

**Recommendations:**
Highlight the background color and make the foreground text in light color or vice-versa.
or,
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to clearly see the text written inside the controls.

**MAS Reference:**
MAS1.4.3: [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
**Attachment For Reference:**
![accessibility a11y_vscode_langmode_colorcontrast](https://user-images.githubusercontent.com/39204909/41697781-134d54c2-7538-11e8-900d-a25c26d97c0a.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
Duplicate #51831
